specs/interop/predeploys.md

```diff
@@ -94,19 +94,21 @@ The `ExecutingMessage` event represents an executing message. It MUST be emitted
 to `executeMessage`.
 
 ```solidity
-event ExecutingMessage(bytes,bytes);
+event ExecutingMessage(bytes32,bytes);
```

Comment:
nit but let's name the arguments in this event definition for clarity/explicitness